vochain/vochaininfo/vochaininfo.go

```diff
@@ -68,14 +72,17 @@ func (vi *VochainInfo) updateCounters() {
 	mempoolSize.Set(uint64(vi.vnode.MempoolSize()))
 }
 
-// Height returns the current number of blocks of the blockchain
+// Height returns the current number of blocks of the blockchain.
 func (vi *VochainInfo) Height() uint64 {
 	return height.Get()
 }
 
 // BlockTimes returns the average block time in milliseconds for 1, 10, 60, 360 and 1440 minutes.
-// Value 0 means there is not yet an average
+// Value 0 means there is not yet an average.
 func (vi *VochainInfo) BlockTimes() *[5]uint64 {
+	if vi.vnode.IsSynchronizing() {
+		return &[5]uint64{defaultBlockTime, 0, 0, 0, 0}
```

Comment:
i'd avoid changing BlockTimes, which is called in many places. pushed a commit

Reply:
Umm, but if the chain is synchronizing the blocktimes will have completely absurd values (such as 0.005). So having a fallback for such scenario that returns a kind of correct value, makes sense IMHO.